src/pages/_layouts/FocussedLayout.vue

```diff
@@ -0,0 +1,13 @@
+<script setup lang="ts">
+import FocussedLayout from '@/components/layouts/FocussedLayout.vue';
+</script>
+
+<template>
+  <FocussedLayout>
```

Comment:
why they have same names? maybe name component as `CFocussedLayout` (C - refers to 'component')?

Reply:
its always a pain, when there are vue files with same name